style.css

```diff
@@ -1,624 +1,995 @@
-    gap: 24px;
-    padding: 14px;
-    width: 100%;
-    height: 100%;
-  }
+.navbar > a {
+  text-decoration-line: none;
+  color: #fff;
+}
 
-  .section-title {
-    margin-top: 34px;
-    font-size: 40px;
-    font-weight: 400;
-    font-family: 'Poppins', sans-serif;
-    line-height: 52px;
-    order: 1;
-    color: #1c1a19;
-    justify-content: center;
-    text-align: center;
-  }
+.hero {
+  display: grid;
+  grid-template-columns: 60% 40%;
+  grid-template-rows: 180px 60px 165px 185px;
+  gap: 10px;
+  width: 100%;
+  height: 590px;
+}
 
-  .horizontal-line {
-    margin-top: 24px;
-    width: 100%;
-    border: 1px solid #212121;
-  }
+.hero-title {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 56px;
+  font-weight: 400;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  width: 375px;
+  padding: 60px 24px 0;
+}
 
-  .works-outer-wrapper {
-    display: flex;
-    justify-content: center;
-    width: 100%;
-    height: 20%;
-    order: 3;
-    position: relative;
-    margin-top: 80px;
-  }
+.hero-sub-title {
+  font-family: 'Roboto', sans-serif;
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 32px;
+  padding: 10px 24px;
+  width: 350px;
+}
 
-  .first-card {
-    width: 100%;
-  }
+.hero-description {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  width: 350px;
+  padding-left: 24px;
+}
 
-  .first-work {
-    width: 100%;
+.media-icon {
+  display: flex;
+  align-items: flex-start;
+  padding-inline-start: 0;
+  list-style: none;
+  margin-left: 24px;
+}
+
+.media-icon li {
+  padding-right: 20px;
+}
+
+.hero-image {
+  grid-row: 1/span 4;
+  grid-column: 2/span 1;
+  background-image: url("./images/group-bg-img.png");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  background-size: 120%;
+  transform: translate(-6.5%, 5%);
+}
+
+.tools-used-here li {
+  padding: 8px;
+  background-color: #999;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  flex-grow: 0;
+  color: #fff;
+  height: 40px;
+  margin-right: 10px;
+  border: 1px solid #3d3797;
+}
+
+.first-project-tools li {
+  padding: 10px 12px;
+  border: 1px solid #8993a4;
+  height: 40px;
+}
+
+ul.skill li {
+  padding: 15px;
+  margin-left: -50px;
+  text-align: start;
+}
+
+#social-media li {
+  padding-right: 10px;
+}
+
+.recent-works-section {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  max-width: 375px;
+  background-color: #fff;
+  color: black;
+  padding: 10px 24px;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+.recent-works-title {
+  width: 100%;
+  border-bottom: 1px solid #888;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  flex-grow: 0;
+  margin: 0;
+}
+
+.recent-works-title h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+}
```

Comment:
- [ ] Please fix the styles in each project card because currently, they look overriding 😃 
![image](https://user-images.githubusercontent.com/90222110/200009829-88e08865-cb53-4a21-bbad-92993b480fea.png)

Reply:
![fix-style-issue](https://user-images.githubusercontent.com/32777408/200088654-6278e930-80c7-4ffd-b0de-31d820fd0dae.png)

This issue has been resolved @BertrandConxy 

I have attached an image of the resolved issue for your kind perusal